components/builder-jobsrv/src/migrations/jobs.rs

```diff
@@ -354,5 +354,17 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                     "#,
     )?;
 
+    // Get a list of all the cancel pending jobs
+    migrator.migrate(
+        "jobsrv",
+        r#"CREATE OR REPLACE FUNCTION get_cancel_pending_jobs_v1()
+                     RETURNS SETOF jobs
+                     LANGUAGE SQL VOLATILE AS $$
```

Comment:
This can probably be `STABLE` instead of `VOLATILE`, no?

Reply:
I think this does need to be volatile.. the set of cancel pending jobs changes, and the function does not return the correct set if marked stable.

Reply:
Ahh, interesting.  I thought `VOLATILE` was just for insert/update/delete.  TIL.